doc/cla/corporate/dynapps.md

```diff
@@ -8,11 +8,11 @@ declaration.
 
 Signed,
 
-Stijn Houben stijn.houben@dynapps.be https://github.com/stijnh92
+Pieter Paulussen pieter_paulussen@me.com https://github.com/PieterPaulussen
```

Comment:
Are you sure you don't want to do the other way around? Use your corporate email for your corporate CLA. You can manage multiple emails per github account if you wish at https://github.com/settings/emails